tests/Doctrine/Tests/ORM/Functional/SecondLevelCacheQueryCacheTest.php

```diff
@@ -1099,6 +1099,21 @@ public function testCacheablePartialQueryException()
             ->getResult();
     }
 
+    /**
+     * @expectedException \Doctrine\ORM\Cache\CacheException
+     * @expectedExceptionMessage Second level cache does not support partial entities.
+     */
+    public function testCacheableForcePartialLoadHintQueryException()
+    {
+        $this->evictRegions();
+        $this->loadFixturesCountries();
+
+        $this->_em->createQuery("SELECT c FROM Doctrine\Tests\Models\Cache\Country c")
+            ->setCacheable(true)
+            ->setHint(Query::HINT_FORCE_PARTIAL_LOAD, true)
+            ->getResult();
+    }
```

Comment:
`@expectedException` and `@expectedExceptionMessage` have been deprecated in PHPUnit 8.x and removed [in PHPUnit v9.0](https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-9.0.md#removed). We should avoid using them.

We might need to import `Doctrine\ORM\Cache\CacheException` but this should be the way to go:

```suggestion
    public function testCacheableForcePartialLoadHintQueryException()
    {
        $this->evictRegions();
        $this->loadFixturesCountries();
        
        $this->expectedException(CacheException::class);
        $this->expectedExceptionMessage('Second level cache does not support partial entities.');

        $this->_em->createQuery("SELECT c FROM Doctrine\Tests\Models\Cache\Country c")
            ->setCacheable(true)
            ->setHint(Query::HINT_FORCE_PARTIAL_LOAD, true)
            ->getResult();
    }
```

Reply:
99 occurrences of this in tests on 2.7, i move this to a mainteance task for 2.8

Reply:
99 occurrences of this in tests on 2.7, i move this to a mainteance task for 2.8